Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -1323,6 +1323,10 @@ bool isInitialValueForLonghand(CSSPropertyID longhand, const CSSValue& value)
                 return true;
         }
         break;
+    case CSSPropertyOffsetPosition:
+        if (isValueID(value, CSSValueNormal))
+            return true;
+        break;
```

Comment:
Could you change `initialValueForLonghand` instead of changing this? Notably this needs to be changed: https://searchfox.org/wubkat/rev/f61c5111898cf9fd9ee106240f085c69776dc8a8/Source/WebCore/css/parser/CSSPropertyParser.cpp#904

Reply:
Right fixed. We were actually failing a test because of this but it wasn't obvious since we were failing it previously as well. So maybe the coverage is ok here.